doc/source/whatsnew/v1.2.0.rst

```diff
@@ -607,6 +607,7 @@ Datetimelike
 - Bug in :meth:`Series.isin` with ``datetime64[ns]`` dtype and :meth:`.DatetimeIndex.isin` failing to consider timezone-aware and timezone-naive datetimes as always different (:issue:`35728`)
 - Bug in :meth:`Series.isin` with ``PeriodDtype`` dtype and :meth:`PeriodIndex.isin` failing to consider arguments with different ``PeriodDtype`` as always different (:issue:`37528`)
 - Bug in :class:`Period` constructor now correctly handles nanoseconds in the ``value`` argument (:issue:`34621` and :issue:`17053`)
+- Bug in :meth:`DataFrame.first` and :meth:`Series.first` returning two months for offset one month when first day is last calendar day (:issue:`29623`)
```

Comment:
so we have three options

1. include in 1.2
2. change to 1.3
3. hold-off merging for a week or so, and then include in 1.2.1

i'm ok with 2 or 3, not so keen on 1.

Reply:
> 3\. hold-off merging for a week or so

actually, no harm in adding a 1.2.1 whatsnew now.

Reply:
I am fine with either one of them

Reply:
ok let's push on 1.3

Reply:
Done